components/openapi/src/main/scala/pl/touk/nussknacker/openapi/http/backend/LoggingAndCollectingSttpBackend.scala

```diff
@@ -3,34 +3,31 @@ package pl.touk.nussknacker.openapi.http.backend
 import com.typesafe.scalalogging.Logger
 import org.slf4j.LoggerFactory
 import pl.touk.nussknacker.engine.api.test.InvocationCollectors.{CollectableAction, ServiceInvocationCollector, TransmissionNames}
-import pl.touk.nussknacker.openapi.http.backend.LoggingAndCollectingSttpBackend.HttpBackend
-import sttp.client.monad.MonadError
-import sttp.client.ws.WebSocketResponse
-import sttp.client._
+import sttp.capabilities.Effect
+import sttp.client3._
+import sttp.model.ResponseMetadata
 
 import java.nio.charset.StandardCharsets
 import java.util.UUID
 import scala.concurrent.{ExecutionContext, Future}
 
 object LoggingAndCollectingSttpBackend {
 
-  type HttpBackend = SttpBackend[Future, Nothing, Nothing]
-
   private case class WithStringResponseBody[T](original: T, stringResponseBody: () => Option[String])
 
   private val transmissionNames = TransmissionNames("request", "response")
 
 }
 
-class LoggingAndCollectingSttpBackend(delegate: HttpBackend, baseLoggerName: String)
-                                     (implicit ec: ExecutionContext, serviceInvocationCollector: ServiceInvocationCollector) extends HttpBackend {
+class LoggingAndCollectingSttpBackend[+P](delegate: SttpBackend[Future, P], baseLoggerName: String)
```

Comment:
indent! ;)

Reply:
Good catch :D

Reply:
It was general notice, can you check other places? For example `handleResponse` in the same file... :P

Reply:
Checked and fixed, looks like this was the last one